src/pybind/mgr/cephadm/templates/services/prometheus/prometheus.yml.j2

```diff
@@ -36,6 +38,10 @@ scrape_configs:
     tls_config:
       ca_file: mgr_prometheus_cert.pem
     honor_labels: true
+    relabel_configs:
+    - source_labels: [__address__]
+      target_label: cluster
+      replacement: {{ cluster_fsid }}
```

Comment:
This section assumes you are using secure communication. Is this the case? what security implications has this new feature? are we taking them into account? have we did any security assessment for the impact?